### DISCUSSION TITLE
how to change the host

### DISCUSSION THREAD

**Discussion by liuxsen** [verdaccio/verdaccio#2302] [Q&A]
![image](https://user-images.githubusercontent.com/12598696/122908225-78298f00-d386-11eb-89c0-b7817593dc2c.png)

this is my config.yaml

![image](https://user-images.githubusercontent.com/12598696/122908465-bb83fd80-d386-11eb-9907-6677482cbfd4.png)

**Comment by juanpicado** [comment]
Maybe you can be more descriptive, what do you want to change and how?  Provide examples please
